scripts/README.md.jinja2

```diff
@@ -0,0 +1,39 @@
+<!-- This document is automatically generated. Do not edit by hand! -->
+# {{ name }}
+
+{{ description | wordwrap(100, break_on_hyphens=False) }}
+
+{% if inputs %}
+## Inputs
+
+| Name | Description | Required | Default |
+|------|-------------|----------|---------|
+{% for key, input in inputs.items() -%}
+| `{{ key }}` | {{ input.description }} | {{ input.required | default('False') }} | {{ "`" ~ input.default ~ "`" if
+input.default is defined else '' }}|
```

Comment:
These line breaks are from autoformatting in my editor. This line is a bit unwieldy. Maybe it's worth preprocessing this input in the script before passing it to the template.

Reply:
Most editors support editorconfig. We could add one of those to the root of this repository and then add a rule for jinja2 configuring an excessive line length.